src/main/kotlin/com/terraformation/backend/tracking/api/ObservationsController.kt

```diff
@@ -149,6 +149,25 @@ class ObservationsController(
     )
   }
 
+  @ApiResponseSimpleSuccess
+  @Operation(summary = "Stores the results of a completed observation of a plot.")
+  @PostMapping("/ad-hoc/monitoring")
```

Comment:
A minor naming thing we should probably discuss as a group: should this be `adHoc` instead of `ad-hoc`? We use camel case for multi-word path elements elsewhere in the API.